lib/browser/guest-view-manager.js

```diff
@@ -314,34 +303,6 @@ const watchEmbedder = function (embedder) {
     }
   }
   embedder.on('-window-visibility-change', onVisibilityChange)
-
-  const destroyEvents = ['will-destroy', 'crashed', 'did-navigate']
```

Comment:
Is it okay to not care about the `crashed` event ?

Reply:
The lifetime  of guest WebContents is now automatically managed by the embedder iframe, when the embedder iframe is gone because crashing or navigation, the guest WebContents will be automatically destroyed.

We still listen to `will-destroy` of embedder, because if we manually destroy embedder, the embedder will be destroyed earlier than the guests, and the embedder will become invalid when calling `detachGuest` later in guest's `destroyed` event.

Reply:
Awesome, thanks for the clarification!